ocl/load-concepts-to-ocl.sh

```diff
@@ -208,7 +239,8 @@ export_concepts_to_json() {
 
   cp ${SDK_DIR}/${PROJECT_NAME}.sql local/
   export USE_GOLD_MAPPINGS=1
-  ./sql-to-json.sh local/${PROJECT_NAME}.sql PIH PIH staging
+#  ./sql-to-json.sh local/${PROJECT_NAME}.sql PIH PIH staging
+  ./sql-to-json.sh local/${PROJECT_NAME}.sql PIH PIH production
```

Comment:
Similar to the above, I would just leave this as staging.  If we ever run this again, it will be for a different use case, and we are going to want to run it against a staging server first.